CHANGELOG.md

```diff
@@ -1,6 +1,378 @@
 Changelog
 =========
 
+0.6.0
+-----
+
+**Date:** 2021-12-XX
+
+C++14, New Solvers, I/O via openPMD API, HIP Support
```

Comment:
I was doubting how to call those "solvers". Was considering "extended numerics" instead, but thought that was possibly confusing wrt floating-point arithmetic. Also, we internally call those things solvers, however that may be not obvious for everyone